packages/airnode-node/src/evm/handlers/process-transactions.ts

```diff
@@ -58,16 +58,7 @@ export async function processTransactions(
   // =================================================================
   // STEP 4: Submit transactions for each wallet
   // =================================================================
-  const receipts = await fulfillments.submit(state3);
-  const successfulReceipts = receipts.filter((receipt) => !!receipt.data);
-  successfulReceipts.forEach((receipt) => {
-    logger.info(`Transaction:${receipt.data!.hash} submitted for Request:${receipt.id}`, baseLogOptions);
```

Comment:
I really like the receipts refactor and the usage of the new `applyFulfillment()` fuction but I'm not finding where this info log entry was moved and I wanted to check that we are still displaying it since I find it useful.

Reply:
It was moved here
https://github.com/api3dao/airnode/pull/734/files#diff-bc443c624cfa9d6ffa22e623859ab1c2923d39f98f02559744469483a234a4dcR123